app/src/app/[lng]/[inventory]/InventoryResultTab/EmissionsForecast/EmissionsForecastChart.tsx

```diff
@@ -38,29 +47,33 @@ export const EmissionsForecastChart = ({
   const convertToLineChartData = (
     forecastData: EmissionsForecastData,
   ): LineChartData[] => {
-    const sectors = Object.keys(
+    let sectors = Object.keys(
```

Comment:
Why the change to `let` here? It seems like it's not being changed afterwards...